### ISSUE TITLE
add validation set

### ISSUE THREAD

**Issue by kaczmarj** [jhajagos/LSTMSeq2ML#20]
We need to add a validation set. We have at least two options.

1. split training set into train/validation sets. there can be a user-defined parameter for the split ratio and a random seed to control 
2. add a validation set in the hdf5 file

we will go with option 1 in the short term because it will take less time and effort. but it would be nice to implement 2.
